compiler/ballerina-backend-jvm/src/main/ballerina/compiler_backend_jvm/jvm_method_gen.bal

```diff
@@ -457,15 +457,17 @@ function generateLambdaMethod(bir:AsyncCall callIns, jvm:ClassWriter cw, string
     mv.visitTypeInsn(CHECKCAST, STRAND);
 
     // load and cast param values
-    bir:BType[] paramBTypes = []; 
+    bir:BType?[] paramBTypes = [];
     int paramIndex = 1;
     foreach var paramType in paramTypes {
-        mv.visitVarInsn(ALOAD, 0);
-        mv.visitIntInsn(BIPUSH, paramIndex);
-        mv.visitInsn(AALOAD);
-        checkCastFromObject(paramType.typeValue, mv);
-        paramBTypes[paramIndex -1] = paramType.typeValue;
-        paramIndex += 1;
+        if (paramType is bir:VarRef) {
```

Comment:
Rather than having `is` checks in all places, shall we move this to a helper method and use that?

We have followed similar approach in other places as well. Refer the methods at the end of `jvm_method_gen.bal`

Reply:
Ack. Will modify these.